### ISSUE TITLE
Isherwood Farms location is not unique.

### ISSUE THREAD

**Issue by freijie** [CleverRaven/Cataclysm-DDA#50800]
### Describe the bug

Isherwood Farms location is generated more than once per world, with Isherwood family npcs getting duplicated.

### Steps To Reproduce

Make a world, travel a lot.

### Expected behavior

This location should be unique.

As a side note, it would be nice to improve outline of the map chunk to make it blend in the world properly:
Change forest type to match the rest of overmap (instead of "#" type, è type)
Remove some of the outer forest tiles to avoid making it look like a perfect square

### Versions and configuration

- OS: W10
- Game Version: c850e91
- Graphics version: [Tiles or Terminal]
- Ingame language: English
- Mods loaded: magiclysm, some unrelated to mapgen mods.

**Comment by RenechCDDA** [comment]
This is intentional to some degree, and there are some plans on how to 'fix' it. See also: #49322

tl;dr: It would be bad if you could lock yourself out from finding a special location by not correctly guessing the direction from your original spawn. Therefore, if you go *really far* you can find more than one.

**Comment by actual-nh** [comment]
> As a side note, it would be nice to improve outline of the map chunk to make it blend in the world properly:
> Change forest type to match the rest of overmap (instead of "#" type, è type)
> Remove some of the outer forest tiles to avoid making it look like a perfect square

If you could make this into a feature request?

**Comment by freijie** [comment]
> 
> 
> This is intentional to some degree, and there are some plans on how to 'fix' it. See also: #49322
> 
> tl;dr: It would be bad if you could lock yourself out from finding a special location by not correctly guessing the direction from your original spawn. Therefore, if you go _really far_ you can find more than one.

In my current run I have 3 Isherwood farms already
on a dingle town with dist. between towns = 8

Good to know its a known issue, I'll make a feature request later.
